File: server/models/order_model.js
const db = require("../db/db");
const orderSchema = new db.mongoose.Schema(
  {

    user_id: {
      type: db.mongoose.Schema.Types.ObjectId,
      required: true,
      ref: "user"
    },
    oder_date: {
      type: String,
      require: true
    },
    total_price: {
      type: Number,
      require: true
    },
    status: {
      type: String,
      require: true
    },
    payment_method: {
      type: String,
      require: true
    },
    delivery_address: {
      type: String,
      require: true
    },
    shipping_fee: {
      type: String,
      require: true
    },
    carrier_id: {
      type: db.mongoose.Schema.Types.ObjectId,
      required: false,
      ref: "carrier"
    },
    code: {
      type: String,
      required: true
    }
  },
  {
    collection: "order",
    timestamps: true,
    versionKey: false
  }
);
const orderModel = db.mongoose.model("order", orderSchema);
module.exports = { orderModel };

File: app/src/main/java/com/example/petify/data/server/enitities/OrderModel.kt
package com.example.petify.data.server.enitities

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class OrderModel(
    @field:SerializedName("_id") var id: String = "",
    @field:SerializedName("user_id") var userId: String = "",
    @field:SerializedName("oder_date") var oderDate: String = "",
    @field:SerializedName("total_price") var totalPrice: Double = 0.0,
    @field:SerializedName("status") var status: String = "",
    @field:SerializedName("payment_method") var paymentMethod: String = "",
    @field:SerializedName("delivery_address") var deliveryAddress: String = "",
    @field:SerializedName("shipping_fee") var shippingFee: Double = 0.0,
    @field:SerializedName("carrier_id") var carrierId: String = "",
    @field:SerializedName("code") var code: String = ""
): Serializable